nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessGroupResource.java

```diff
@@ -1470,42 +1437,41 @@ public Response getRemoteProcessGroups(
      * Creates a new connection.
      *
      * @param httpServletRequest request
-     * @param groupId The group id
-     * @param connectionEntity A connectionEntity.
+     * @param groupId            The group id
+     * @param connectionEntity   A connectionEntity.
      * @return A connectionEntity.
      */
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     @Path("{id}/connections")
-    // TODO - @PreAuthorize("hasRole('ROLE_DFM')")
     @ApiOperation(
-        value = "Creates a connection",
-        response = ConnectionEntity.class,
-        authorizations = {
-            @Authorization(value = "Data Flow Manager", type = "ROLE_DFM")
-        }
+            value = "Creates a connection",
+            response = ConnectionEntity.class,
+            authorizations = {
+                    @Authorization(value = "Write - /process-groups/{uuid}", type = "")
```

Comment:
In the code it looks like we also check WRITE access to the source and destination, should the docs here reflect that?

```
source.authorize(authorizer, RequestAction.WRITE, NiFiUserUtils.getNiFiUser());
destination.authorize(authorizer, RequestAction.WRITE, NiFiUserUtils.getNiFiUser());
```